list.c

```diff
@@ -39,16 +39,16 @@ void vListInitialise( List_t * const pxList )
 	/* The list structure contains a list item which is used to mark the
 	end of the list.  To initialise the list the list end is inserted
 	as the only list entry. */
-	pxList->pxIndex = ( ListItem_t * ) &( pxList->xListEnd );			/*lint !e826 !e740 !e9087 The mini list structure is used as the list end to save RAM.  This is checked and valid. */
+	pxList->pxIndex = &( pxList->xListEnd );			/*lint !e826 !e740 !e9087 The mini list structure is used as the list end to save RAM.  This is checked and valid. */

```

Comment:
A cast was removed here, which suggests that one of the lint error codes should probably be removed. But I don’t know what linting tool FreeRTOS uses or where to find a list of the codes, so I don’t know which one to remove. Could someone in the know suggest please?

Reply:
I can take care of the lint comment (we are actually switching to Coverity anyway).

Reply:
Great! In that case I have no other concerns myself, this just needs reviewing.